### PR TITLE
feat(Generator): generate a 2-menu list app with test example

### PR DESCRIPTION
**Pull request by jsomsanith**
**Please check if the PR fulfills these requirements**

- [x] The commit(s) message(s) follows our
  [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**
`yo talend:react-cmf` generates an app that
* display 2 menus (2 routes)
* each route fetch a static json and display a list via `HomeListView`
* contains an action creator snap test and a container snap test (mocking CMF)

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://162.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://162.talend.surge.sh)

File: packages/generator/generators/react-cmf/templates/src/app/components/App.container.js [hunk 1]
```diff
@@ -1,21 +1,23 @@
...
-	<div>
-		<IconsProvider />
-		<Notification />
-		{props.children}
-	</div>
-);
 App.propTypes = {
-	//store: React.PropTypes.object.isRequired,
+	children: React.PropTypes.element,
+	// store: React.PropTypes.object.isRequired,
```
Comment (talend-glorieux) [inline packages/generator/generators/react-cmf/templates/src/app/components/App.container.js]: Please remove commented code.

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://162.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://162.talend.surge.sh)

**Pull request merged**